release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -112,8 +112,10 @@ and xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp
 ==== Automatic rotation of certificates
 
 Automated service CA rotation will be available in this release in a future
-z-stream update. This is to allow time for administrators to plan accordingly
-for their environments.
+z-stream update. In previous versions, the service CA did not automatically
+renew, leading to service disruption and requiring manual intervention. With the
+service CA and signing key auto-rotating before expiration, administrators can
```

Comment:
Consider rewording as:
The service CA and signing key auto-rotates before expiration. This allows administrators to plan for their environments in advance, avoiding service disruption.